### PR TITLE
Power user toggle and request button

### PR DESCRIPTION
**Pull request by H4mzaCode**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,8 +25,10 @@
     "electron": "^25.1.1"
   },
   "dependencies": {
+    "axios": "^1.4.0",
     "better-sqlite3": "^8.4.0",
     "bootstrap": "^5.3.0",
+    "electron-dl": "^3.5.0",
```
Comment (jonathanmajh) [inline package.json:31]: are these new dependencies needed?

File: renderer/item_main.html [hunk 2]
```diff
@@ -141,6 +353,7 @@ <h5 class="modal-title" id="confirmModalLabel">Confirm Upload</h5>
         </div>
         <h1 class="col">Item Standardizer</h1>
         <div class="col d-flex align-items-center justify-content-end">
+          <i id="secret-button" data-clicks='0' class="material-icons pointer md-size" style="text-align: right; padding-right: 20px;" >question_mark</i>
```
Comment (jonathanmajh) [inline renderer/item_main.html:356]: button is visible

File: renderer/item_main.html [hunk 3]
```diff
@@ -15,6 +15,218 @@
...
+                      <label for="manu-name">Preferred Manufacturer</label>
+                    </div>
+                    <input type="text" class="form-control" id="pref-manu" style="display: none" placeholder="Manufacturer Name">
+                  </div>
+                </div>
+
+                <div class="row">
+                  <div class="col-md">
+                    <div class="form-floating input-group">
+                      <input type="text" class="form-control" id="part-num" required>
```
Comment (jonathanmajh) [inline renderer/item_main.html:192]: should only be required when a manufacturer is specified

File: renderer/item_main.js [hunk 4]
```diff
@@ -21,6 +24,192 @@ window.onload = function() {
...
+    let manu = document.getElementById("manu-name");
+    let pref = document.getElementById("pref-manu");
+    let part = document.getElementById("part-num");
+    let asset = document.getElementById("asset-num");
+    let web = document.getElementById("web-link");
+    
+    let mail = makeTextFile(
+        `<textarea id="textbox" cols="2" rows="13" style="display: none">
+To: Maximo Item request <maximo.item@iko.com>
+Subject: Item request
```
Comment (jonathanmajh) [inline renderer/item_main.js:122]: include some identifier in the subject (maybe timestamp?)

File: package.json (hunk 1 above)
Comment (H4mzaCode) [reply package.json:31]: We need the electron-dl one but we don't need axios I'll remove that one

File: renderer/item_main.html [hunk 5]
```diff
@@ -15,6 +15,218 @@
...
+                    </div>
+                  </div>
+                </div>
+
+                <div class="row">
+                  <div class="col-md">
+                    <div class="form-floating input-group">
+                      <select class="form-select" id="manu-name">
+                        <option value="Unknown">Unknown</option>
+                        <option value="Generic">Generic</option>
```
[the remark this answers is not in the document]
Comment (H4mzaCode) [reply renderer/item_main.html:219]: Low key that's kinda what it is already. If it's generic they just select generic, but if it isn't then they can specify it by writing it out once they select "other"

**Pull request merged**